QUEST.tsv

```diff
@@ -2098,156 +2098,156 @@ QUEST_20150714_002097	But, Raius is the disciple I acknowledge.{nl}His passion t
+QUEST_20150714_002116	$When you find all three spatial magic stones, go to the fusion device again and place all the magic stones there. {nl}The path to the master's research will then open at the Raudi Building.
+QUEST_20150714_002117	$Spirit
+QUEST_20150714_002118	$Please… save us from this horrible contract… {nl}Remove us from this… contract of declaration…
+QUEST_20150714_002119	$The contract of declaration… {nl}It expires as we speak… I have no regrets…
+QUEST_20150714_002120	$Eternal imprisonment… from the restraints of the contract… {nl}Break… the pot of silence…
+QUEST_20150714_002121	$The pot of silence is at Memum Corridor… {nl}I'm so… tired…
+QUEST_20150714_002122	$Where to start…
+QUEST_20150714_002123	$I was a disciple of Agailla Flurry when I was alive. {nl}I didn't have any magical abilities so I couldn't use magic.
+QUEST_20150714_002124	$That's why I survived Helgasercle's wrath. {nl}He must've thought he didn't have to kill me since I wasn't able to use magic.
+QUEST_20150714_002125	$I am confident in my knowledge of terms and theories more than the other disciples. {nl}That's why I studied magic. And while I was studying, I got to know the magic stone of interaction.
+QUEST_20150714_002126	$I came here when Helgasercle attacked Mage Tower. {nl}I thought that, if I could use the magic stone of interaction, I'd be able to use magic.
+QUEST_20150714_002127	$But, Akon chased me from Mage Tower. {nl}Due to this disturbance, the magic stone activated defensive magic and my and Akon's spirit were linked.
+QUEST_20150714_002128	$As a result, Akon was sealed and my body disappeared without a trace. {nl}We became shackled to each other.
+QUEST_20150714_002129	$I've been tied here for hundreds of years… {nl}Since I can't do anything with my astral body.
+QUEST_20150714_002130	$The bigger problem is that Akon was released as well. {nl}I lost the body, but Akon, who is a demon, is still alive.
+QUEST_20150714_002131	$Memorial Stone
+QUEST_20150714_002132	$By avoiding the eyes of Helgasercle, I was able to come here to continue my research. {nl}I needed to find a way: a way to reduce the power of demons and increase that magical power.
+QUEST_20150714_002133	$By my name, Balryus, the one who walks the way of Linker, I swear that I will not surrender to the demons.
+QUEST_20150714_002134	$I've made many magical tools, but I wasn't able to repel the demon attacks. {nl}Without checking out the result, I close my eyes.
+QUEST_20150714_002135	$The Records of Investigation
+QUEST_20150714_002136	$The basics of Link Magic are to find the common factors in each others' spirits. {nl}So it is impossible if the receiver does not possess any magical abilities or bases. {nl}
+QUEST_20150714_002137	$I've completed the first stage magic stone. {nl}It still does not satisfy the previous objective of delivering the perfect magic and magical power somewhat, {nl}but now at last it is possible to accept Link Magic even without magical ability.{nl}
```

Comment:
where does the "at last" come from? Also, you changed "acceptor" to "receiver" above, you should change "accept" to something like "be on the receiving end of". Link Magic or Linker Magic?